src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -1242,6 +1257,51 @@ def cli_cosmosdb_network_rule_list(client, resource_group_name, account_name):
     cosmos_db_account = client.get(resource_group_name, account_name)
     return cosmos_db_account.virtual_network_rules
 
+def cli_cosmosdb_identity_show(client, resource_group_name, account_name):
+    """ Show the identity associated with a Cosmos DB account """
+
+    cosmos_db_account = client.get(resource_group_name, account_name)
+    return cosmos_db_account.identity
+
+def cli_cosmosdb_identity_assign(client,
+                                 resource_group_name,
+                                 account_name):
+    """ Show the identity associated with a Cosmos DB account """
+
+    existing = client.get(resource_group_name, account_name)
+
+    if 'SystemAssigned' in existing.identity.type:
+        return existing.identity
+
+    if existing.identity.type == 'UserAssigned':
```

Comment:
In which way can customer set the identity type `UserAssigned`? `cosmosdb identity assign` doesn't seem to take `--assign-type` parameter, it can only add `SystemAssigned` identity...